ios/goodtags.xcodeproj/project.pbxproj

```diff
@@ -336,6 +336,7 @@
 			buildConfigurationList = 13B07F931A680F5B00A75B9A /* Build configuration list for PBXNativeTarget "goodtags" */;
 			buildPhases = (
 				743A12DB8A85CC8909793C41 /* [CP] Check Pods Manifest.lock */,
+				368F4735A18E08EAEB68A451 /* Download Search DB */,
```

Comment:
Note I don't have a macOS machine to run this on, so my changes to this file were fully hand crafted; if you could test this by deleting the `src/assets/generated_db/` directory, running a build from within Xcode, and then verifying that directory is created and populated, I'd be grateful 🙏